lms/djangoapps/course_goals/management/commands/goal_reminder_email.py

```diff
@@ -134,6 +153,13 @@ def handle(self, *args, **options):
             self._handle_all_goals()
         except BaseException:  # pylint: disable=broad-except
             log.exception("Error while sending course goals emails: ")
+            tracker.emit(
+                'edx.course.goal.email.failed',
+                {
+                    'timestamp': datetime.now(),
+                    'reason': 'base exception',
```

Comment:
You can get details from base exception (I am not sure if this would be helpful in this scenario)

except BaseException as exc:
'reason': str(exc)